docs/source/en/using-diffusers/reusing_seeds.mdx

```diff
@@ -10,7 +10,57 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
 specific language governing permissions and limitations under the License.
 -->
 
-# Improve image quality with deterministic generation
+# Improve image quality from a pipeline
```

Comment:
Hmm is `re-using seeds` the correct file here? Should we maybe just make a new file "Control Brightness of Images"?

Reply:
I think it might be nice to collect all the tips for improving image quality in one doc (I can rename the file`reusing_seeds`) which may be more convenient for users especially if they want to apply both tips (generate a deterministic batch + apply zero SNR).  But I'm also happy to create a new file if you prefer!

Reply:
I think it would be better to create a new file because "re-using" seeds is a very common use case on its own (e.g. a user generates 8 images, same prompt, but different seed. 1 image is very nice. The user now wants to re-use the seed of this image and slightly change the prompt, ..." 

So I'd prefer adding a new "Controlling Brightness" doc file. @sayakpaul wdyt?

Reply:
In agreement 👍

Reply:
Done! 😄